olive/model.py

```diff
@@ -750,11 +811,11 @@ def __init__(
         version: Optional[int] = None,
     ):
         super().__init__(
+            framework=Framework.ONNX,
+            model_file_format=ModelFileFormat.ONNX,
             model_path=None,
             name=name,
             version=version,
-            is_file=False,
-            is_aml_model=False,
```

Comment:
specify the model_storage_kind to folder?

Reply:
Done. It doesn't really matter since this is a dummy value. The actual values are the storage_kind of the component.